src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMNavigator.java

```diff
@@ -99,10 +99,28 @@
      */
     private HashSet<String> navigatorProjects = new HashSet<>();
 
+    /**
+     * To store if project owner is group
+     */
+    private boolean isGroup;
+
+    /**
+     * To store the full name of the owner
+     */
+    private String ownerName = "";
```

Comment:
Nononono 😆

the projectOwner is the ownerName.

Reply:
No, project owner is username. But the payload doesn't provide username, it only provides full name of user (owner).